tests/linearGaussian_snl_test.py

```diff
@@ -170,7 +169,7 @@ def test_snl_on_linearGaussian_c2st(num_dim: int, prior_str: str, set_seed):
 
 
 @pytest.mark.slow
-def test_multi_round_snl_on_linearGaussian_c2st(set_seed):
+def test_c2st_multi_round_snl_on_linearGaussian(set_seed):
```

Comment:
In all fairness, the substantive test functions test _both_ API and functionality, i.e. the API tests are _onlyAPI_ tests. Just a comment between us.